db/models/Starship.js

```diff
@@ -0,0 +1,59 @@
+import mongoose from "mongoose";
+const { Schema, model, models } = mongoose;
+
+// {
+//     name: "CR90 corvette",
+//     model: "CR90 corvette",
+//     manufacturer: "Corellian Engineering Corporation",
+//     default_cost_in_credits: 3500000,
+//     max_atmosphering_speed: "950",
+//     passengers: "600",
+//     cargo_capacity: "3000000",
+//     starship_class: "corvette",
+//     img: "https://static.wikia.nocookie.net/starwars/images/2/24/TantiveIV-TSWB.png/revision/latest?cb=20221006055135",
+//   },
+
```

Comment:
Please remove this comment 🧹
Some arguments why we won't keep comments like this one:
- Comments containing code or data structures can be misleading. Developers might wonder why the commented-out code is there, whether it's old code, temporary code, or some reference for future use.
- Large commented-out blocks can interrupt the flow and reduce the readability of the code. 
-  Leaving commented-out code can give an impression of carelessness or that the codebase is not well-maintained.